public/icons/README.md

```diff
@@ -0,0 +1,9 @@
+# Icon Usage Guidelines
+
+This folder contains SVG icons that are used in the design system. When using icons from this folder, please follow the guidelines below:
+
+- Icons located in public/ folder is the current source of truth. All icons will be placed in this folder. Ensure that icons placed gere are using the `class` attribute, and not `className` for SVG element. This is necessary for the icons to work properly when the folder is copied over using the "dapple"-setup. 
```

Comment:
```suggestion
- Icons located in public/ folder is the current source of truth. All icons will be placed in this folder. Ensure that icons placed gere are using the `class` attribute, and not `className` for SVG element. 
```

I do not think it should be relevant to mention Dapple at this level of documentation:

As I see it this is a general problem that just has not surfaced yet because 90% of all work (and very little) design has been going on in the CMS up until now.